### PR TITLE
Lf 276 farm data permissions

### PR DESCRIPTION
**Pull request by kaavyalakshmanan**
- Created migration file to revoke permissions for workers
- Added middleware to routes
- Minor changes to controller
- Wrote tests

### CODE REVIEW

File: packages/api/src/controllers/userFarmDataController.js [hunk 1]
```diff
@@ -18,37 +18,44 @@ const Knex = require('knex');
...
         const farm_id = req.params.farm_id;
-        const data = await knex('farmDataSchedule').where({ farm_id, is_processed: false }).returning('*');
-        res.status(200).send(data);
+        const result = await userFarmModel.query().where('farm_id', farm_id);
+        if (!result.length) {
+          res.sendStatus(404)
+        }
+        else {
+          res.status(200).send(result);
+        }
```
Comment (kikemarquez8) [inline packages/api/src/controllers/userFarmDataController.js:51]: I don't think this is right, this is supposed to be fetching farmDataSchedule data. not userFarm

File: packages/api/tests/userFarmData.test.js [hunk 2]
```diff
@@ -0,0 +1,220 @@
+/*
```
Comment (kikemarquez8) [inline packages/api/tests/userFarmData.test.js:1]: please format this file, 2 spaces instead of tabs for indentation

**Review by kikemarquez8 — CHANGES_REQUESTED** [review]

File: packages/api/src/controllers/userFarmDataController.js (hunk 1 above)
Comment (kaavyalakshmanan) [reply packages/api/src/controllers/userFarmDataController.js]: @kikemarquez8 done

File: packages/api/tests/userFarmData.test.js (hunk 2 above)
Comment (kaavyalakshmanan) [reply packages/api/tests/userFarmData.test.js:1]: @kikemarquez8 done

**Review by kikemarquez8 — APPROVED** [review]

**Pull request merged**